pytest_mpiexec/pytest_plugin.py

```diff
@@ -120,24 +120,22 @@ def mpi_runtest(item):
             )
         except subprocess.TimeoutExpired as e:
             if e.stdout:
-                item.add_report_section(
-                    "mpiexec pytest", "stdout", e.stdout.decode("utf8", "replace")
-                )
+                item.add_report_section("mpiexec pytest", "stdout", e.stdout)
```

Comment:
These changes cause:

```
TypeError: a bytes-like object is required, not 'str'
```

because `e.stdout` is bytes, but `add_report_section` takes `str`.